packages/component-library/stories/Button.story.js

```diff
@@ -3,34 +3,69 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
 import { checkA11y } from "@storybook/addon-a11y";
-import { withKnobs, text } from "@storybook/addon-knobs";
+import { withKnobs, text, color, select, files } from "@storybook/addon-knobs";
+import notes from "./button.notes.md";
 import { Button } from "../src";
 import { storybookStyles } from "./storyStyles";
-
-const displayName = Button.displayName || "Button";
-
-const description = `
-  This is some basic usage with the button with providing a label to show the text.
-  Clicking should trigger an action.`;
-
-const simpleButtonLabel = "Hello button";
+import { file } from "@babel/types";
```

Comment:
Is this needed? Is this needed in every file where we import notes?

Reply:
No, this is required if you are using the file knob.  I experimented with using it in this story, but missed removing this import in the final design.  It's been removed.